CMakePresets.json

```diff
@@ -0,0 +1,75 @@
+{
+    "version": 6,
+    "cmakeMinimumRequired": {
+        "major": 3,
+        "minor": 14,
+        "patch": 0
+      },
+    "configurePresets": [
+        {
+            "name": "common",
+            "hidden": true,
+            "cacheVariables": {
+                "FREECAD_USE_PYBIND11:BOOL": "ON"
+            }
+        },
+        {
+            "name": "debug",
+            "displayName": "Debug",
+            "description": "Default debug profile",
+            "binaryDir": "${sourceDir}/build/debug",
+            "inherits": [
+                "common"
+            ],
+            "cacheVariables": {
+                "CMAKE_BUILD_TYPE": "Debug"
+            }
+        },
+        {
+            "name": "release",
+            "displayName": "Release",
+            "description": "Default release profile",
+            "binaryDir": "${sourceDir}/build/release",
+            "inherits": [
+                "common"
+            ],
+            "cacheVariables": {
+                "CMAKE_BUILD_TYPE": "Release"
+            }
+        },
+        {
+            "name": "conda",
+            "hidden": true,
+            "cmakeExecutable": "${workspaceFolder}/.conda/freecad/bin/cmake",
```

Comment:
```suggestion
```
to get the build to work I had to remove this since it didn't match the actual path of cmake on the windows conda env, but it isn't necessary to specify this anyways since the executable will be included in PATH after running activate

Reply:
I had already thought this could be removed, thanks for the confirmation.